### PR TITLE
backend cleanup fixmes

### PR DESCRIPTION
**Pull request by joka**

### CODE REVIEW

**Comment by nidico** [comment]
Can you please rebase/merge?

File: src/adhocracy_core/adhocracy_core/authentication/__init__.py [hunk 1]
```diff
@@ -103,7 +103,8 @@ def get_tokenmanager(request: Request, **kwargs) -> ITokenManger:
 def _get_raw_x_user_headers(request: Request) -> tuple:
     """Return not validated tuple with the X-User-Path/Token values."""
     user_url = request.headers.get('X-User-Path', '')
-    # FIXME find a proper solution, user_url/path as userid does not work
+    # TODO find a proper solution, useid should just be an identifier.
```
Comment (pallix) [inline src/adhocracy_core/adhocracy_core/authentication/__init__.py]: typo: useid -> userid

File: src/adhocracy_mercator/adhocracy_mercator/sheets/mercator.py [hunk 2]
```diff
@@ -436,7 +432,7 @@ def index_budget(resource: IResource, default) -> str:
     The returned values are the same values as per the "requested_funding"
     field, or "above_50000" if the total budget value is more than 50,000 euro.
     """
-    # FIXME: Why is finance '' in the first pass of that function
+    # TODOE: Why is finance '' in the first pass of that function
```
Comment (pallix) [inline src/adhocracy_mercator/adhocracy_mercator/sheets/mercator.py]: typo

**Comment by pallix** [comment]
When running the protractor acceptance tests locally, there is a lot of tests failing on this branch, specially the one concerning the ratings.

**Comment by joka** [comment]
> When running the protractor acceptance tests locally, there are a lot of tests failing on this branch, specially the one concerning the ratings.

Are these tests stable?

**Comment by pallix** [comment]
When runned locally yes.

**Comment by joka** [comment]
i fixed one regression regarding subscriber registration and renamed some stuff prevent this kind of errors in the future.

All tests pass on my local machine.

src/mercator/tests/acceptance/test_mercator_filter.py did only  pass if run manually, but this is an unrelated issue.

**Pull request merged**